src/docs/product/performance/transaction-summary.mdx

```diff
@@ -87,7 +87,7 @@ Frontend transactions will have a "Web Vitals" tab. Clicking on the tab will tak
 
 ### Key Transaction
 
-If you find an important transaction you want to return to, click "Key Transaction" in the upper right corner to star it. This will be saved so you can view this from your [Performance Homepage](/product/performance/). Keep in mind any key transaction you mark will not be shared with others in your organization and are limited to ten per user.
+If you find an important transaction you want to return to, click "Star for Team" in the upper right corner and mark it as a key transaction for any of your teams. This will be saved so you can view this from your [Performance Homepage](/product/performance/). The key transactions you mark will be shared with members of the same team and each team can have up to 100 key transactions.
```

Comment:
Would this need an early adopters note?

Reply:
That's a good point. Not sure if we have a pattern for this where an existing feature gets a revamp and goes into early adopters. cc @imatwawana if you have any suggestions?

Reply:
Ah, one thing I noticed and forgot to add in my review, can you change the heading on line 88 to **Key Transactions** (plural)?